src/promptflow-tools/tests/test_aoai.py

```diff
@@ -71,6 +72,23 @@ def test_aoai_chat_with_function(
         assert "function_call" in result
         assert result["function_call"]["name"] == "get_current_weather"
 
+    def test_aoai_chat_with_name_in_roles(
```

Comment:
the test cases should be added to `test_common` rather than `test_aoai` as you made changes to `common.py`

Reply:
add more tests to test_common and keep the test_aoai change for e2e tests